superset/charts/post_processing.py

```diff
@@ -328,14 +328,19 @@ def apply_post_process(
         if query["result_format"] not in (rf.value for rf in ChartDataResultFormat):
             raise Exception(f"Result format {query['result_format']} not supported")
 
-        if not query["data"]:
+        data = query["data"]
+
+        if isinstance(data, str):
+            data = data.strip()
```

Comment:
Prevents the `pandas.errors.EmptyDataError: No columns to parse from file` error when `query["data"]` is `\n`—which occurs from the `/api/v1/chart/<pk>/data/?format=csv&type=post_processed` endpoint when the the chart payload contains no data.